### ISSUE TITLE
Do you have a pre-trained model?

### ISSUE THREAD

**Issue by aj96** [Jabb0/FastFlow3D#12]
Do you have the model you trained on waymo available to download, or do we have to download dataset, preprocess data, and train for 3 days to get the model you achieved?

**Comment by Jabb0** [comment]
Hi,
thanks for the interest. 
I do not have a trained model available. Sorry.

If you train it, I suggest considering a smaller batch size as you need a lot of GPU memory (4x Titan X).
But BatchNorm is sensitive to batch size so the results should be worse.
